breathe/renderer/sphinxrenderer.py

```diff
@@ -431,7 +431,33 @@ def set_context(self, context: RenderContext) -> None:
         if self.context.domain == '':
             self.context.domain = self.get_domain()
 
+    # XXX: fix broken links in XML generated by Doxygen when
+    # SEPARATE_MEMBER_PAGES is set to YES; this function should be
+    # harmless when SEPARATE_MEMBER_PAGES is NO!
```

Comment:
I just decided to run Doxygen on gdb to actually understand how it derives those anchors, and added a text explaining it to the function comment. Also I can now see that this might still fail to catch some of the possibilities, although I would rather refactor the code after a few other urgent things are out of the way!

Reply:
@utzig Thanks a lot, this is definitely good for merging now. Should I merge this and release or would you rather have I wait for a refactor in this PR? Can always fixup in a future PR too of course.

Reply:
Thanks, I'd rather refactor it a bit after #560 is done!

Reply:
Any idea when will it be possible to merge those PRs and do a point release?

Reply:
@utzig Hi, sorry for the delay. I intend of checking all the Breathe PRs today and then I'll make a release. Just to make sure: is only #560 ready or is this one ready for merging too?

Reply:
Both work well for me, would be glad if you could include both. Thanks!